content/en/events/2023/kcscn/schedule.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Schedule
+type: docs
+weight: 3
+---
+
+**NOTE:** You **MUST** pick up your badge before attending the summit event.
+
+## Steering AMA
+
+The Summit will start off with an AMA with members of the Kubernetes Steering
+Committee.  Bring any and all questions about how we're running the project,
+ongoing efforts & issues, and whatever you'd like to discuss with Steering.
```

Comment:
I think we won't have this, but just an opening by Kevin (and me?) if I see that correctly from the preliminary schedule in our google doc

Reply:
yes. Updated. 

Currently, our plan is as you said, a welcome/opening by you and Kevin.